src/CoreFx.Private.TestUtilities/src/System/TheoryExtensions.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace System
+{
+    public static class TheoryExtensions
+    {
+        /// <summary>
+        /// Converts an IEnumerable<typeparamref name="T"/> into an Xunit theory compatible enumerable.
+        /// </summary>
+        public static TheoryData ToTheoryData<T>(this IEnumerable<T> data)
```

Comment:
I think we should migrate things there eventually. Let them bake here a bit and move them up (same goes for our Assert extensions, etc.).